libkbfs/disk_block_cache.go

```diff
@@ -138,17 +181,25 @@ func newDiskBlockCacheStandardFromStorage(config diskBlockCacheConfig,
 	startedCh := make(chan struct{})
 	startErrCh := make(chan struct{})
 	cache = &DiskBlockCacheStandard{
-		config:     config,
-		maxBlockID: maxBlockID.Bytes(),
-		tlfCounts:  map[tlf.ID]int{},
-		tlfSizes:   map[tlf.ID]uint64{},
-		log:        log,
-		blockDb:    blockDb,
-		metaDb:     metaDb,
-		tlfDb:      tlfDb,
-		startedCh:  startedCh,
-		startErrCh: startErrCh,
-		shutdownCh: make(chan struct{}),
+		config:           config,
+		maxBlockID:       maxBlockID.Bytes(),
+		tlfCounts:        map[tlf.ID]int{},
+		tlfSizes:         map[tlf.ID]uint64{},
+		hitMeter:         metrics.NewMeter(),
```

Comment:
I guess I could refactor it to use `config.MetricsRegistry()`. Would you prefer that?